pkg/kubelet/token/token_manager_test.go

```diff
@@ -151,6 +152,15 @@ func TestRequiresRefresh(t *testing.T) {
 			exp:           start.Add(60 * time.Minute),
 			expectRefresh: true,
 		},
+		{
+			now:           start.Add(0 * time.Minute),
+			// expiry will be overwritten by the tweak below.
+			exp:           start.Add(60 * time.Minute),
+			expectRefresh: false,
+			requestTweaks: func(tr *authenticationv1.TokenRequest) {
```

Comment:
I don't love this approach to testing, but it also didn't seem like the `nil` case should be among these tests.  I could break this out to its own `func` if you think that would read better, though.